install_config/router/customized_haproxy_router.adoc

```diff
@@ -13,48 +13,666 @@ toc::[]
 
 == Overview
 
-The HAProxy router is based on a
-link:http://golang.org/pkg/text/template/[*golang* template] that
-generates the HAProxy configuration file from a list of routes. If you
-want a customized template router to meet your needs, you can customize
-the template file, build a new container image, and run a customized router.
-Alternatively you can use a xref:../../dev_guide/configmaps.adoc#dev-guide-configmaps[ConfigMap].
-
-One common case for this might be implementing new features within the
-application back ends. For example, it might be desirable in a highly-available
-setup to xref:using-stick-tables[use stick-tables] that synchronizes between
-peers. The router plug-in provides all the facilities necessary to make this
-customization.
+The default haproxy router is intended to satisfy the needs of most users.
+However, it does not expose all of the capability of HAProxy and users
+may need to modify the router for their own needs.  Here is how to go
+about customizing the router.
```

Comment:
Should we remove this last line (Here is how...) and perhaps rename this section to be Customizing the Router?